projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts

```diff
@@ -16,13 +19,20 @@ import {
 export class CaseFileViewFolderComponent implements OnInit, OnDestroy {
 
   private static readonly UNCATEGORISED_DOCUMENTS_TITLE = 'Uncategorised documents';
+  private static readonly DOCUMENT_SEARCH_FORM_CONTROL_NAME = 'documentSearchFormControl';
+  private static readonly MINIMUM_SEARCH_CHARACTERS = 3;
 
   @Input() public categoriesAndDocuments: Observable<CategoriesAndDocuments>;
 
   public nestedTreeControl: NestedTreeControl<DocumentTreeNode>;
   public nestedDataSource: DocumentTreeNode[];
   public categories: CaseFileViewCategory[] = [];
   public categoriesAndDocumentsSubscription: Subscription;
+  public documentFilterFormGroup: FormGroup;
```

Comment:
You can probably get away without using a FormGroup. I can only see a single FormControl, therefore a FormGroup isn't needed.